crates/bevy_core_pipeline/src/debug_gradient.wgsl

```diff
@@ -0,0 +1,48 @@
+#define_import_path bevy_core_pipeline::debug_gradient
+
+// Define a function to map a range of values to a high contrast color gradient
+// optimized for representational clarity.
+// this uses the matplotlib plasma color map.
+
+// Color from the plasma gradient quantized to six colors, for value between
+// 0 and 5, clamped.
+fn plasma_color_at_index(value: f32) -> vec3<f32> {
+    let plasma_colors = array(
+        vec3(0.868,0.941,0.011), // #f0f921
+        vec3(0.966,0.386,0.0326), // #fca636
+        vec3(0.753,0.126,0.121), // #e16462
+        vec3(0.444,0.0188,0.282), // #b12a90
+        vec3(0.144,0.0,0.396), // #6a00a8
+        vec3(0.00142,0.000488,0.245), // #0d0887
+    );
+    // There is no way to index an array in wgsl from a dynamic value,
```

Comment:
Technically, you can, you just need to enable a WGPU feature that may not be supported everywhere. This is better for a debug utility though.